LanPartyHub/GameConfig.xaml.cs

```diff
@@ -86,8 +89,20 @@ private void GameConfigAddSetting(object sender, RoutedEventArgs e)
                 var setting = new KeyValue();
```

Comment:
Use object initialization. new KeyValue{
Key = key,
Value = value
}

Reply:
updated in commit 5687b25